### ISSUE TITLE
Fine-tuning inception_v3  failed with tf0.12 in windows

### ISSUE THREAD

**Issue by ligangcc92** [tensorflow/models#865]
I run the command as the document saied:

python train_image_classifier.py --train_dir="D:\PythonProjects\flowers-model\inception_v3"  --dataset_dir="D:\PythonProjects\\train_data"  --dataset_name=flowers  --dataset_split_name=train  --model_name=inception_v3  --checkpoint_path="D:\PythonProjects\pretrained_checkpoints\inception_v3.ckpt"  --checkpoint_exclude_scopes=InceptionV3/Logits,InceptionV3/AuxLogits/Logits  --trainable_scopes=InceptionV3/Logits,InceptionV3/AuxLogits/Logits 

and get some error: 

InvalidArgumentError (see above for traceback): Assign requires shapes of both tensors to match. lhs shape= [5] rhs shape= [1001]
	 [[Node: save/Assign_8 = Assign[T=DT_FLOAT, _class=["loc:@InceptionV3/AuxLogits/Conv2d_2b_1x1/biases"], use_locking=true, validate_shape=true, _device="/job:localhost/replica:0/task:0/gpu:0"](InceptionV3/AuxLogits/Conv2d_2b_1x1/biases, save/RestoreV2_8/_5)]]

what's wrong with that?

**Comment by michaelisard** [comment]
This kind of usage question is better asked on [Stackoverflow](http://stackoverflow.com/questions/tagged/tensorflow).
